docs/Reference/CLI/Subcommands/Slashing-Protection.md

```diff
@@ -60,6 +60,34 @@ Path to the Teku data directory. The default directory is OS-dependent:
 * Unix/Linux: `$XDG_DATA_HOME/teku` if `$XDG_DATA_HOME` is set; otherwise `~/.local/share/teku`
 * Windows: `%localappdata%\teku`.
 
+### data-validator-path
+
+=== "Syntax"
+
+    ```bash
+    teku slashing-protection import --data-validator-path=<PATH>
+    ```
+
+=== "Example"
+
+    ```bash
+    teku slashing-protection import --data-validator-path=/home/me/me_validator
+    ```
+
+=== "Environment variable"
+
+    ```bash
+    TEKU_DATA_VALIDATOR_PATH=/home/me/me_validator
+    ```
+
+=== "Configuration file"
+
+    ```bash
+    data-validator-path: "/home/me/me_validator"
+    ```
+
+Path to the validator client data. The default is `<data-base-path>/validator` where `<data-base-path>` is specified using [`--data-base-path`](#data-base-path-data-path).
```

Comment:
So the context of why this path is needed is because slashing protection data is stored under this folder, in a folder called `slashprotection`, and the data imported will be written to this folder. We've had other tickets previously regarding the usage being confusing, and I'm guessing maybe it's this context that is missing?